packages/@okta/vuepress-site/docs/guides/configure-native-sso/main/index.md

```diff
@@ -0,0 +1,440 @@
+
+To generate a new set of tokens:
+
+* Use Auth Code with PKCE to obtain the authorization code for the first client.
+* Exchange the code for tokens.
+* Exchange the existing tokens from client 1 for new tokens for client 2.
+
+In this example, you want to SSO to multiple apps that are created by the same company. Each client represents one app, and you can register multiple clients for SSO. When a user signs in to one app, all the other apps that are registered are also automatically signed in.
+
+### Use Authorization Code with PKCE to obtain the authorization code for client 1
+
+Provide the `device_sso`, `openid`, and `offline_access` scopes in the first request to the `/authorize` endpoint using the Authorization Code with PKCE flow. See [Use the Authorization Code flow with PKCE](/docs/guides/implement-auth-code-pkce/use-flow/) for information on the parameters that are being passed in this request.
+
+**Example Authorization Code with PKCE request**
+
+```
+  https://${yourOktaDomain}/oauth2/default/v1/authorize?client_id=0oa10sqibtVca6tme1d7&response_type=code&scope=openid device_sso offline_access&redirect_uri={configured_redirect_uri}&state=state-8600b31f-52d1-4dca-987c-386e3d8967e9&code_challenge_method=S256&code_challenge=qjrzSW9gMiUgpUvqgEPE4_-8swvyCtfOVvg55o5S_es
+```
+
+The user is prompted to provide their credentials. After the authorization server verifies those credentials, the authorization code is sent to the `redirect_uri` that you specified. The following is an example of the authorization code returned.
+
+**Example response**
+
+```
+  https://{configured_redirect_uri}/?code=S_NuB0TNeDMXD_5SKZO6FuXFOi_J9XB-sHAk0Dc0txQ&state=state-8600b31f-52d1-4dca-987c-386e3d8967e9
+```
+
+### Exchange the code for tokens
+
+To exchange the authorization code for tokens, pass the code to your authorization server's `/token` endpoint along with the `code_verifier` that was generated. See [Exchange the code for tokens](/docs/guides/implement-auth-code-pkce/exchange-code-token/) for information on the parameters that are being passed in this request.
+
+**Example request**
+
+```bash
+curl --location --request POST \
+  --url https://${yourOktaDomain}/oauth2/default/v1/token \
+  --header 'accept: application/json' \
+  --header 'cache-control: no-cache' \
+  --header 'content-type: application/x-www-form-urlencoded' \
+  --data 'grant_type=authorization_code&client_id={clientId}&redirect_uri=yourApp%3A%2Fcallback&code=CKA9Utz2GkWlsrmnqehz&code_verifier=M25iVXpKU3puUjFaYWg3T1NDTDQtcW1ROUY5YXlwalNoc0hhakxifmZHag'
```

Comment:
Be consistent with placeholder values, here we don't provide the real client id but we do in the other example above.

Reply:
```suggestion
  --data 'grant_type=authorization_code&client_id={clientId}&redirect_uri={configuredRedirectUri}&code=CKA9Utz2GkWlsrmnqehz&code_verifier=M25iVXpKU3puUjFaYWg3T1NDTDQtcW1ROUY5YXlwalNoc0hhakxifmZHag'
```